packages/realm-server/server.ts

```diff
@@ -22,7 +23,8 @@ export function createRealmServer(realms: Realm[]) {
     }
 
     res.on("finish", () => {
-      console.log(`${req.method} ${req.url}: ${res.statusCode}`);
+      log.info(`${req.method} ${req.url}: ${res.statusCode}`);
+      log.debug(JSON.stringify(req.headers));
```

Comment:
could we do a named logger here? i think this is useful for logging in hosted env, but it can be overwhelming for dev and test